### PR TITLE
Update to pulp-container 2.8.3

### PR DESCRIPTION
**Pull request by newswangerd**
Issue: AAH-1188

### CODE REVIEW

**Review by awcrosby — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -82,7 +82,6 @@ jobs:
       matrix:
         env:
           - TEST: pulp
-          - TEST: azure
```
Comment (fao89) [inline .github/workflows/ci.yml:85]: why removing azure tests?

Comment (newswangerd) [reply .github/workflows/ci.yml:85]: @fao89 I just ran the plugin template. I'm not sure why it did that

Comment (fao89) [reply .github/workflows/ci.yml:85]: I think your plugin-template is outdated

Comment (newswangerd) [reply .github/workflows/ci.yml]: looks like it was. Good catch

**Pull request merged**